core/src/main/java/org/jdbi/v3/GeneratedKeys.java

```diff
@@ -13,13 +13,14 @@
  */
 package org.jdbi.v3;
 
+import com.fasterxml.classmate.ResolvedType;
+import org.jdbi.v3.exceptions.ResultSetException;
+import org.jdbi.v3.tweak.ResultSetMapper;
+
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
 
-import org.jdbi.v3.exceptions.ResultSetException;
-import org.jdbi.v3.tweak.ResultSetMapper;
```

Comment:
Is there a good way to specify it for both?

This is what I have right now:

![screen shot 2015-12-10 at 4 57 17 pm](https://cloud.githubusercontent.com/assets/129097/11733098/2247228e-9f5f-11e5-97da-490d318c4f66.png)